tests/basic-test.sh

```diff
@@ -450,16 +450,20 @@ echo sudoers2 >tree-C/etc/sudoers
 $OSTREE commit ${COMMIT_ARGS} -b test3-C2 -s "C" --tree=dir=tree-C
```

Comment:
Should this whole hunk be part of the previous commit where we added this test?

Reply:
You're right, fixed in #1645